src/jdk.javadoc/share/legal/jquery.md

```diff
@@ -1,9 +1,9 @@
-## jQuery v3.5.1
+## jQuery v3.6.0
 
 ### jQuery License
 ```
 jQuery v 3.5.1
```

Comment:
Does this line need to be changed as well?

Reply:
Thanks for catching this, fixed in new commit.